app/models/miq_group.rb

```diff
@@ -4,7 +4,8 @@ class MiqGroup < ApplicationRecord
   TENANT_GROUP = "tenant"
 
   belongs_to :tenant
-  belongs_to :miq_user_role
+  has_one    :entitlement
```

Comment:
Yup

On Tuesday, March 15, 2016, Keenan Brock <notifications@github.com> wrote:

> In app/models/miq_group.rb
> <https://github.com/ManageIQ/manageiq/pull/7302#discussion_r56227929>:
>
> > @@ -4,7 +4,8 @@ class MiqGroup < ApplicationRecord
> >    TENANT_GROUP = "tenant"
> >
> >    belongs_to :tenant
> > -  belongs_to :miq_user_role
> > +  has_one    :entitlement
>
> safe to assume that this will change to has_many in the future?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/7302/files/508b45fbd2565184bc7b18bd3bf41255b050ec81#r56227929>
>


-- 
Chris Arcand
@chrisarcand